reladomo-scala-common/src/main/scala/com/folio_sec/reladomo/scala_api/package.scala

```diff
@@ -26,4 +26,6 @@ package object scala_api extends OperationImplicits with AggregateDataImplicits
 
   type Timestamp = java.sql.Timestamp
 
+  type BigDecimal = java.math.BigDecimal
```

Comment:
I'm not sure if having a type alias for `java.math.BigDecimal` is really useful. As for `Timestamp`, Reladomo (Java) heavily uses `java.sql.Timestamp`, so I thought that having an alias for it is reasonable.

Reply:
`reladomo-scala-common` is a library which is used runtime. You should not have a type alias which is needed only for the code generator.

Reply:
Thank you.

I confirmed https://github.com/laysakura/reladomo-scala/pull/1/files#diff-b7d7f2dc639b2ac1018db156121ee8c7R514 solves the problem.